template.yaml

```
@@ -417,6 +417,31 @@ Resources:
       Tags:
         - Key: Name
           Value: OrchSessionTable
+      ResourcePolicy:
+        PolicyDocument:
+          Version: 2012-10-17
+          Statement:
+            - Sid: AllowOrchsessionTableAccessForLogout
```

Not a blocker but I think this follows how we generally name an `Sid`
```suggestion
            - Sid: AllowOrchSessionTableCrossAccountDeleteAccess
```